science/imager/Portfile

```diff
@@ -0,0 +1,98 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           compilers 1.0
+PortGroup           active_variants 1.1
+
+name                imager
+version             202211
+set my_version      [string tolower [clock format [clock scan 2000-[string range ${version} 4 5]-10] -format %b]][string range ${version} 2 3][string range ${version} 6 end]
+supported_archs     arm64 x86_64
+set my_machine      [if {[string match *arm* ${os.arch}]} {list arm64} {list x86_64}]
```

Comment:
`os.arch` refers to the machine doing the build, which is not necessarily the architecture for which the build is being done. The latter is in `configure.build_arch` (when a universal build is not being done; if this port wants to support a universal variant, additional work is needed).